Mathlib/Algebra/Category/Ring/Adjunctions.lean

```diff
@@ -92,6 +92,10 @@ instance : HasForget₂ CommRingCat CommMonCat where
   forget₂ := { obj M := .of M, map f := CommMonCat.ofHom f.hom }
   forget_comp := rfl
 
+/- #adaptation_note
+This apparently became slower in nightly-2025-02-12. (Possibly due to changes in `simp +arith`?)
+-/
```

Comment:
This way it becomes easier to grep for all adaptation notes about a give nightly/issue/...
```suggestion
/- #adaptation_note nightly-2025-02-12
This apparently became slower. (Possibly due to changes in `simp +arith`?) -/
```